frontend/src/renderer/util/autoSortStandards.svelte

```diff
@@ -78,9 +79,8 @@
 
             for(let i = 0; i < matchingStandards.length; i++) {
                 each(matchingStandards[i], function (string) {
-                    if(includes(image.name.toLowerCase(), string.toLowerCase())) {
-                        image[probabilityScoreProperties[i]]++;
-                    }
+                    const count = split(image.name.toLowerCase(), string.toLowerCase()).length - 1;
+                    image[probabilityScoreProperties[i]] += count;
                 });
```

Comment:
yes dumb I know but this fixes it (and removes a line of code so fight me

Reply:
🥊🥊

Reply:
🥊🥊